README.md

```diff
@@ -6,18 +6,23 @@
 + Parse JSON to get clean API data
 
 ## Intro
-use twitter API to make GET and POST requests to create tweets, view tweets
-
+For this lab we will be using the GitHub API to create and view Gists. In this lab you will create a UI that allows users to see a list of their gists and be able to create new ones.
 
 ## Instructions
-
-build methods `myTweets();` return current user's 5 most recent tweets
-
-`createTweet("tweet content")` to send a tweet
-
-`viewTweets("username")` to view a specific user's tweets
-
-build a frontend with 3 forms to trigger API calls to display data in the browser
-
-needs tests
-<p data-visibility='hidden'>View <a href='https://learn.co/lessons/js-apis-lab' title='Twitter API'>Twitter API</a> on Learn.co and start learning to code for free.</p>
+You will need to visit https://github.com/settings/tokens and create a personal token to use. This token allows you to make request data for your account.
+
+- Create the method `myGists(username, token)` that returns the gists for a user.
```

Comment:
need to provide students what API endpoint to use  and how to use it.

https://developer.github.com/v3/gists/#list-a-users-gists